### PR TITLE
Signup/Login

### PR DESCRIPTION
**Pull request by kevin-06-huang**
### What this PR does (required):
- alter the colour of the SignUp/Login buttons as well as the text in theme and useStyles for login and signup to bring in line with mockup
- alter the style (bold, font) of the login/signup page and form to bring in line with mockup
- alter the parameter of onSubmit and Formik in SignUpForm and SignUp (username to name) to bring in line with mockup
- rebuild the textfield instead with components in order to use FormLabel instead of InputLabel to bring the look of SignUpForm and LoginForm in line with mockup

### Screenshots / Videos (front-end only):
SignUp:Login frontend demo

### Any information needed to test this feature (required):
- After merging in the request, the change should be visible right away. Just run the server and client like usual.

### Any issues with the current functionality (optional):

### CODE REVIEW

File: client/src/pages/Login/LoginForm/LoginForm.tsx [hunk 1]
```diff
@@ -46,44 +50,38 @@ export default function Login({ handleSubmit }: Props): JSX.Element {
...
-            type="password"
-            autoComplete="current-password"
-            helperText={touched.password ? errors.password : ''}
-            error={touched.password && Boolean(errors.password)}
-            value={values.password}
-            onChange={handleChange}
-          />
+          <FormControl id="email" fullWidth={true} margin="dense">
+            <FormLabel>
+              <Typography className={classes.label}>EMAIL ADDRESS</Typography>
```
Comment (ugur-akin) [inline client/src/pages/Login/LoginForm/LoginForm.tsx:55]: We should avoid capital casing content in JSX, and use CSS for it instead for accesibilty reasons (more reading on this [here](https://medium.com/@mandy.michael/why-you-shouldnt-write-your-content-in-uppercase-instead-use-css-b03ac2c65b99)).

Please apply wherever appropriate 👍🏼

File: client/src/pages/Login/LoginForm/LoginForm.tsx [hunk 2]
```diff
@@ -46,44 +50,38 @@ export default function Login({ handleSubmit }: Props): JSX.Element {
...
+              className={classes.inputs}
+              name="password"
+              autoComplete="current-password"
+              type="password"
+              error={touched.password && Boolean(errors.password)}
+              value={values.password}
+              placeholder="Your password"
+              onChange={handleChange}
+            />
+            <FormHelperText>{touched.email ? errors.email : ''}</FormHelperText>
```
Comment (ugur-akin) [inline client/src/pages/Login/LoginForm/LoginForm.tsx:83]: Did you mean `password` 😉 ?

File: client/src/pages/SignUp/SignUpForm/SignUpForm.tsx [hunk 3]
```diff
@@ -51,65 +54,57 @@ const SignUpForm = ({ handleSubmit }: Props): JSX.Element => {
...
+            <FormHelperText>{touched.name ? errors.name : ''}</FormHelperText>
+          </FormControl>
+          <FormControl id="password" fullWidth={true} margin="dense">
+            <FormLabel>
+              <Typography className={classes.label}>PASSWORD</Typography>
+            </FormLabel>
+            <OutlinedInput
+              className={classes.inputs}
+              name="password"
+              autoComplete="current-password"
```
Comment (ugur-akin) [inline client/src/pages/SignUp/SignUpForm/SignUpForm.tsx:95]: `new-password` is more appropriate for this (you can check [here](https://developer.mozilla.org/en-US/docs/Web/HTML/Attributes/autocomplete) for the entire list of values accepted)👍🏼

**Review by ugur-akin — COMMENTED** [review]
Good job on this, good use of mui/formik 🚀 It would have been nice to see the style in the mocks matched closer, however. For example, we could wrap the forms in a `Paper` as seen in the mocks and move the "aside text" (e.g. "Already have an account? Login") onto the page itself.

File: client/src/pages/Login/LoginForm/LoginForm.tsx (hunk 2 above)
Comment (kevin-06-huang) [reply client/src/pages/Login/LoginForm/LoginForm.tsx:83]: Yes, my mistake.

File: client/src/pages/SignUp/SignUpForm/SignUpForm.tsx (hunk 3 above)
Comment (kevin-06-huang) [reply client/src/pages/SignUp/SignUpForm/SignUpForm.tsx:95]: Thank you, much appreciated.

File: client/src/pages/Login/LoginForm/LoginForm.tsx (hunk 1 above)
Comment (kevin-06-huang) [reply client/src/pages/Login/LoginForm/LoginForm.tsx:55]: Understood.

**Comment by kevin-06-huang** [comment]
All issues have been addressed.

**Pull request merged**